sonic-xcvrd/scripts/xcvrd

```diff
@@ -979,11 +982,11 @@ class sfp_state_update_task:
 
         logger.log_info("Stop SFP monitoring loop")
 
-    def task_run(self):
+    def task_run(self, sfp_error_event):
         if self.task_stopping_event.is_set():
             return
 
-        self.task_process = multiprocessing.Process(target=self.task_worker,args=(self.task_stopping_event,))
+        self.task_process = multiprocessing.Process(target=self.task_worker,args=(self.task_stopping_event,sfp_error_event))
```

Comment:
Add a space between `,` and `sfp_error_event`

Reply:
fixed